### DISCUSSION TITLE
关于rcore可视化debug的方案

### DISCUSSION THREAD

**Discussion by YXalix** [LearningOS/rust-based-os-comp2022#195] [Q&A]
目前我采用的方法是在os目录下运行make debug，进入gdb，但是对于原生的gdb调试非常陌生，效率非常低。。
1. 尝试过使用gdb-dashboard，但是好像riscv64-unknow-elf-gdb并不支持dashboard，对于gdb-dashboard，有大佬尝试成功的吗？
2. 或者有其他的debug方案嘛
救救孩子:cry

**Comment by Ayana-chan** [comment]
坚决而狂暴地使用println，可以找出99.99%的bug，而且rust对象的输出比c友好多了
